### PR TITLE
feat: Support local node

### PR DESCRIPTION
**Pull request by Sneh1999**
**What changed? Why?**
Adding support for local anvil node for faster testing

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: template/contracts/README.md [hunk 1]
```diff
@@ -14,9 +14,15 @@ Use at your own risk, acknowledging the inherent risks of smart contract technol
 # Contents
 
 - [Introduction](#introduction)
+  - [Contracts](#contracts)
+    - [BuyMeACoffee](#buymeacoffee)
+    - [CustomERC1155](#customerc1155)
+    - [SignatureMintERC721](#signatureminterc721)
+    - [AllolistNFT](#allowlistnft)
```
Comment (pop-punk) [inline template/contracts/README.md:21]:
```suggestion
    - [AllowlistNFT](#allowlistnft)
```

Comment (Sneh1999) [reply template/contracts/README.md]: Thank you!

**Review by cnasc — APPROVED** [review]
LGTM! :shipit:

**Comment by Zizzamia** [comment]
Alright, merging this now! @robpolak and @awilliams1-cb feel free still to share your notes, so we can cover them in a follow up PR.

**Pull request merged**